packages/jest-message-util/src/__tests__/messages.test.js

```diff
@@ -23,6 +23,27 @@ const unixStackTrace =
   at Object.<anonymous> (../jest-jasmine2/build/jasmine-pit.js:35:32)
   at attemptAsync (../jest-jasmine2/build/jasmine-2.4.1.js:1919:24)`;
 
+const assertionStack =
+  '  ' +
+  `
+    Expected value to be of type:
+      "number"
+    Received:
+      ""
+    type:
+      "string"
+
+      at Object.it (__tests__/test.js:8:14)
+      at Object.asyncFn (node_modules/jest-jasmine2/build/jasmine_async.js:124:345)
+      at resolve (node_modules/jest-jasmine2/build/queue_runner.js:46:12)
+          at Promise (<anonymous>)
+      at mapper (node_modules/jest-jasmine2/build/queue_runner.js:34:499)
+      at promise.then (node_modules/jest-jasmine2/build/queue_runner.js:74:39)
+          at <anonymous>
+      at process._tickCallback (internal/process/next_tick.js:188:7)
+      at internal/process/next_tick.js:188:7
```

Comment:
this last one I just saw in the integration tests. It should be the same as the one above it. Ideas?

Reply:
Seems to come from the remaining replace in `cleanupStackTrace` in `integration_tests/utils.js`.  Not sure how to handle it

Reply:
If we really want to do it, I can move this `next_tick` part down there, no reason to have it in `message-util`

Reply:
Figured it out, this is node 4 missing the calling function's name.